packages/react/scripts/fix-files.js

```diff
@@ -0,0 +1,146 @@
+/**
+ * Copyright (c) 2023, WSO2 LLC. (https://www.wso2.com). All Rights Reserved.
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+const fs = require('fs');
+const path = require('path');
+const saas = require('node-sass');
+const {logger} = require('@oxygen-ui/logger');
+
+module.exports = fixFiles = () => {
+  //All files and directories in the src/components folder.
+  const componentFiles = fs.readdirSync(path.resolve(__dirname, '..', 'src/components'));
+
+  // All directories in the src/components folder.
+  const componentDirs = componentFiles.filter(file => {
+    const componentDir = path.resolve(__dirname, '..', 'src/components', file);
+    return fs.lstatSync(componentDir).isDirectory();
+  });
+
+  // The directories containing the files bundled by Rollup.
+  const moduleDirs = ['cjs', 'esm'];
+
+  // The non-component directories in the dist folder.
+  const nonComponentDirsInDist = fs.readdirSync(path.resolve(__dirname, '..', 'dist')).filter(file => {
+    return (
+      !componentDirs.includes(file) &&
+      fs.lstatSync(path.resolve(__dirname, '..', 'dist', file)).isDirectory() &&
+      !moduleDirs.includes(file)
+    );
+  });
+
+  /**
+   * This method fixes the references to the non-component directories in the dist folder such as utils, constants, etc.
+   * in imports.
+   *
+   * @param {string} file - The file to be fixed.
+   *
```

Comment:
Do we really need a space here. https://tsdoc.org/pages/tags/returns/

Reply:
Resolved by https://github.com/wso2/oxygen-ui/pull/67/commits/0e01ce8265da34cda086c64e374195b9cc0684a4